### PR TITLE
update NEO drivers to support DG2 and fix package name for rhel8

### PR DESCRIPTION
**Pull request by dtrawins**
### Details:
 - latest public yum and apt drivers with support for DG2, tested also with iGPU
 - corrected the core package name for rhel8
This script is to be used while building docker images with OV

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
@dtrawins please, create PRs from your own forked repo in the future.

File: scripts/install_dependencies/install_openvino_dependencies.sh [hunk 1]
```diff
@@ -151,7 +151,7 @@ elif [ "$os" == "ubuntu20.04" ] || [ "$os" == "ubuntu21.10" ] || [ "$os" == "ubu
 
 elif [ "$os" == "rhel8" ] ; then
 
-    pkgs_core=(libtbb2 libpugixml1v5)
+    pkgs_core=(tbb.x86_64 pugixml.x86_64)
     pkgs_opencv_req=(gtk3)
     pkgs_python=(python38 python38-pip)
     pkgs_dev=(
```
Comment (ilya-lavrenov) [inline scripts/install_dependencies/install_openvino_dependencies.sh:157]: do we also need to install json-devel for samples? BA requires nlohmann json library, which is supposed to be installed from the system repositories.

Comment (ilya-lavrenov) [reply scripts/install_dependencies/install_openvino_dependencies.sh:157]: BTW, does RHEL8 have 64-bits version of zlib-devel package?

Comment (ilya-lavrenov) [reply scripts/install_dependencies/install_openvino_dependencies.sh:157]: why do we need x86_64 explicitly? such file can be used for aarch64 RHEL as well..

Comment (dtrawins) [reply scripts/install_dependencies/install_openvino_dependencies.sh:189]: @ilya-lavrenov now the platform requirement is removed

**Review by ilya-lavrenov — APPROVED** [review]

**Review by artanokhov — APPROVED** [review]

**Pull request merged**